### ISSUE TITLE
Is it OK to have accessibility: true without a Keyboard Help Dialog?

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#568]
From https://github.com/phetsims/phet-io/issues/1457#issuecomment-595959777:

> @zepumph and I discussed the CCK aspect, that it is supposed to use a11y engine to support deleting circuit elements, but without showing the keyboard help dialog. It's unclear to me whether more work is necessary for that.

**Comment by zepumph** [comment]
This is a question for a11y designers. So I'll assign to them.

@terracoda and @emily-phet, in Circuit Construction Kit there is a way to delete an element or circuit connection by clicking an icon. A PDOM related listener was also added to support using the "delete" key to achieve this. Does this warrant a keyboard help dialog? In pairing with @samreid I wasn't sure what the requirements were for having one. It feels totally fine to me to not have one for this single key.

**Comment by terracoda** [comment]
@zepumph, I think that listening for the delete key is a very good idea - a very intuitive shortcut. I can't be sure if their won't be a conflict with some screen reader software.

Is your question more about if we would want to add instructions about this to the keyboard shortcuts dialog?

**Comment by zepumph** [comment]
>Is your question more about if we would want to add instructions about this to the keyboard shortcuts dialog?

Yes exactly. Since CCK (at least right now) doesn't have any other description or alternative input other than the delete key, I don't worry about the interference with AT. I am wondering if we need to add a keyboard help dialog for this single keyboard interaction.

**Comment by emily-phet** [comment]
@zepumph - thoughts:
1) we don't have the bandwidth to participate in adding a keyboard dialog to CCK, so if a11y folks are needed, it needs to be deferred. 
2) if a11y folks are not needed, then I think it's up to the CCK team if they think adding a keyboard shortcut key will support the needs of that sim.

**Comment by terracoda** [comment]
@zepumph, I agree with @emily-phet, and for something so intuitive it is possible that putting it in the teacher tips could suffice if the team chooses not to make a shortcuts dialog.

**Comment by arouinfar** [comment]
This sim does not at all support alternative input (yet). I think adding a keyboard dialog could potentially cue that the sim has functionality that it does not, especially if the "`Tab` to get started" and "Basic Actions" sections of the dialog are required components. 

We currently describe the delete interaction in the Teacher Tips:
![image](https://user-images.githubusercontent.com/8419308/76883824-5582ff00-6842-11ea-8384-943d3ca05c56.png)

**Comment by zepumph** [comment]
Then I personally don't think that a help dialog is needed.

**Comment by samreid** [comment]
@zepumph do we need to do anything else to close this issue?

**Comment by zepumph** [comment]
I think we're good.
